test.py

```diff
@@ -1,4 +1,4 @@
-import urllib.request
```

Comment:
Thrupthi, Please change the import using from... *. Dont import only specific class.